charts/kubearchive/templates/operator/sink_filters.yaml

```
@@ -0,0 +1,4 @@
+apiVersion: v1
```

You are missing the YAML header.

```suggestion
---
apiVersion: v1
```

---

Fixed.